### DISCUSSION TITLE
InAppPurchases in managed workflow in SDK 43?

### DISCUSSION THREAD

**Discussion by psnet** [expo/expo#14592] [Archived: Questions] [answered]
Can we expect InAppPurchases module to be in managed expo workflow in SDK 43?

**Comment by brentvatne** [comment, marked as answer]
it's possible in SDK 42 with EAS Build. here is a thread about it: https://forums.expo.dev/t/has-anyone-successfully-adopted-iap-with-eas-build-for-their-managed-app/54047
